react-native-template-pytorch-live/template/models/requirements.txt

```diff
@@ -1,5 +1,5 @@
 torch==1.10.0
 torchvision==0.11.1
 torchaudio==0.10.0
-transformers==4.6.1
-scipy==1.7.1
+transformers==4.11.3
```

Comment:
@aaronespasa the Distilbert NLP Q&A example will break if the `transformers` dependency is upgraded to `4.11.3` and needs additional debugging (see screenshot)

![Simulator Screen Shot - iPhone 12 - 2021-12-12 at 20 19 41](https://user-images.githubusercontent.com/489051/145751884-1fa2793e-f949-414a-9318-600636a031a9.png)